src/containers/GradebookPage/index.jsx

```diff
@@ -68,6 +68,7 @@ const mapStateToProps = (state, ownProps) => (
     uploadSuccess: !!(state.grades.bulkManagement &&
                       state.grades.bulkManagement.uploadSuccess),
     showBulkManagement: stateHasMastersTrack(state),
+    showDownloadButtons: stateHasMastersTrack(state),
```

Comment:
I couldn't decide whether to add a second, somewhat redundant prop or to  just use the `showBulkManagement` prop for both purposes (perhaps with a more generic name). Thinking it through again, I don't expect that there will be a course where `showBulkManagement` and `showDownloadButtons` would not agree, so maybe a single prop would be better. Any thoughts either way?

Reply:
I prefer it the way you have it in the PR, but can appreciate why it might give you pause. In my mind this way is more explicit, and enables the possibility for a user of the Gradebook component to not have these two distinct flags coincide. It doesn't really matter for this particular use-case, because the Gradebook component is really tightly coupled to its use on this page, but it's still better to orient ourselves towards reusable components whenever possible (since it helps us get the most out of React).